azure-pipelines.vrt-pr.yml

```diff
@@ -132,17 +110,6 @@ jobs:
           vrTestPackageName: '@fluentui/vr-tests'
           vrTestPackagePath: 'apps/vr-tests'
 
-      - bash: |
-          set -exuo pipefail
-          yarn vr-app create-policy --nonBlockingPipelines '{"$(pipelineId)":{"pipelineStatus": "PENDING","pipelineName": "$(pipelineName)"}}' --clientType 'FLUENTUI'
-        displayName: VR App - Create Policy
-        env:
-          VR_APP_API_URL: $(VR_APP_API_URL)
-          TENANT_ID: $(TenantId)
-          PRINCIPAL_CLIENT_ID: $(PrincipalClientId)
-          SERVICE_CONNECTION_ID: $(ServiceConnectionId)
-          SYSTEM_ACCESSTOKEN: $(System.AccessToken)
-
```

Comment:
wondering if we could move there also the `displayName: 'Run fluentui-screenshotdiff'` step ?